integrations/platform_sh.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "Platform.sh",
+  "image": "https://platform.sh/logos/redesign/Platformsh_logo_black.svg",
+  "auth": {
+    "authorizationURL": "https://accounts.platform.sh/oauth2/authorize",
+    "tokenURL": "https://accounts.platform.sh/oauth2/token",
```

Comment:
```suggestion
    "authorizationURL": "https://auth.api.platform.sh/oauth2/authorize",
    "tokenURL": "https://auth.api.platform.sh/oauth2/token",
```
since 29/4/2020